data/categories.txt

```diff
@@ -3698,6 +3698,68 @@ sk:Zberné suroviny
 sw:Chombo cha kutia taka
 fa:مرکز بازیافت
 
+amenity-recycling_container
+en:Recycling container|U+1F6AE|U+267B
+es:Contenedor de reciclaje
+de:Recyclingbehälter
+ru:Контейнер для вторсырья|утиль
```

Comment:
en, ru, then others (copy the order from an older category)

Reply:
done